src/main/java/api/utils/CategoryUtils.java

```diff
@@ -16,4 +16,15 @@ static Category generateDefaultCategory() {
 
         return category;
     }
+
+    static Category generateSpecificCategory() {
```

Comment:
to keep consistency across project, this should be called `generateDefaultCategory()`

Reply:
Removed generateSpecificCategory() method.